packages/terra-time-input/tests/wdio/time-input-spec.js

```diff
@@ -1,363 +1,285 @@
 Terra.describeViewports('Time Input', ['medium'], () => {
-  describe('Default with no time provided', () => {
-    before(() => browser.url('/#/raw/tests/terra-time-input/time-input/time-input/default'));
-
-    Terra.it.validatesElement();
+  it('displays Default with no time provided', () => {
```

Comment:
Should lower case the second word in all the other `it` blocks.

```suggestion
  it('displays default with no time provided', () => {
```